packages/edit-site/src/components/left-sidebar/index.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * Internal dependencies
+ */
+import InserterPanel from './inserter-panel';
+import NavigationPanel from './navigation-panel';
+
+const LeftSidebar = ( { content, setContent } ) => {
+	let leftSidebar = null;
```

Comment:
I don't think we need `leftSidebar` var anymore or `else if`. We simplify this to:

```javascript
if ( content === 'navigation' ) {
    return <NavigationPanel />;
}

if ( content === 'inserter' ) {
    return <InserterPanel closeInserter={ () => setContent( null ) } />;
}

return null;

```

Reply:
Done! I was driven away by the original code 😓 😄